src/components/Button/Button.tsx

```diff
@@ -40,31 +40,40 @@ const Button: FunctionComponent<ButtonProps> = ({
 	size,
 	title,
 	type = 'primary',
-}) => (
-	<button
-		className={classnames(className, 'c-button', {
-			'c-button--active': active,
-			'c-button--auto': autoHeight,
-			'c-button--small': size === 'small',
-			'c-button--block': block,
-			'c-button--icon': icon && !label,
-			[`c-button--${type}`]: type,
-		})}
-		onClick={disabled ? () => {} : onClick}
-		disabled={disabled}
-		aria-label={ariaLabel}
-		title={title}
-	>
-		{children ? (
-			children
-		) : (
-			<div className="c-button__content">
-				{icon && <Icon className="c-button__icon" name={icon} active={active} />}
-				{label && <div className="c-button__label">{label}</div>}
-				{arrow && <Icon className="c-button__icon" name="caret-down" />}
-			</div>
-		)}
-	</button>
-);
+}) => {
+	const handleButtonClick = (evt: MouseEvent<HTMLElement>) => {
+		evt.stopPropagation();
```

Comment:
Gevaarlijk om dit overal te doen, dit is best iets dat je meegeeft aan de onClick handlers van de Button's die je gebruikt in uw Table dan.
Er zijn ook enkele andere components en testen kapot hierdoor waaronder voor Dropdown, ExpandableContainer, ...